README.md

```diff
@@ -31,6 +31,37 @@ This is an open source project which is officially supported by Exasol. For any
 * [Changelog](doc/changes/changelog.md)
 * [Dependencies](dependencies.md)
 
+### Adapter Properties for JDBC-Based Virtual Schemas
```

Comment:
Without wanting to nitpick, we usually have this kind of information in a separate user guide. I guess I can live with the info in the README in this case though.